modules/issue_tracker/README.md

```diff
@@ -1,11 +1,11 @@
 # Issue Tracker
 
 ## Purpose
-The Issues Module allows users to track issues they have with data, or with their LORIS instance itself. A form with pre-defined fields is provided for users to submit issues, and a filter-form gives a sortable and filterable table view of issues viewable by the user.
+The Issues Module allows users to track issues they have with data, or with their LORIS instance itself. A form with pre-defined fields is provided for users to submit issues, upload attachments and a filter-form gives a sortable and filterable table view of issues viewable by the user.
 
 ## Permissions
-- `issue_tracker_reporter` permission allows adding an issue, editing issues created by the user, and commenting on all issues for the site. 
-- `issue_tracker_developer` permission allows to do the same, as well as closing an issue or editing any field of a submitted issue for the site. 
+- `issue_tracker_reporter` permission allows adding an issue, editing issues created by the user, download issue attachments and commenting on all issues for the site. 
```

Comment:
Could you add a quick note here that says "Issues are related to sites. In order to view an issue, a user must have access to the site associated it."

I know it's not really in the scope of this PR but doing so will allow us to close #5668 at the same time.

Reply:
@johnsaigle added.

Reply:
It looks like you reverted to an earlier commit? I don't see the addition and the `issue_tracker_developer` permission information was deleted.

Reply:
@johnsaigle oops my mistake. It's back now.

Reply:
It looks like the note hasn't been added though

Reply:
@johnsaigle It's not supposed to be the following?
```
## Permissions
- `issue_tracker_reporter` permission allows adding an issue, editing issues created by the user, download issue attachments and commenting on all issues for the site.
- `issue_tracker_developer` permission allows to do the same, as well as closing an issue or editing any field of a submitted issue for the site.
```

Reply:
sorry @johnsaigle now the part from @christinerogers should be added